automation/features/step-definitions/staking.steps.js

```diff
@@ -48,23 +68,323 @@ When(/^I click on use maximum button$/, () => {
     reverse: false,
     timeoutMsg: "callout was not found",
   });
-  if (browser.getByTestId("associate-more-tokens-btn").isDisplayed()) {
-    browser.getByTestId("associate-more-tokens-btn").click();
+  if (stakingPage.associateMoreTokensBtn.isDisplayed()) {
+    stakingPage.associateMoreTokensBtn.click();
   } else {
-    browser.getByTestId("associate-tokens-btn").click();
+    stakingPage.associateTokensBtn.click();
   }
   $('[data-testid="associate-radio-wallet"]').click({ force: true });
-  currentAmountOfTokensInWallet = $(".wallet-card__asset-balance").getText();
+  currentAmountOfTokensInWallet = browser
+    .getByTestId("eth-wallet-balance")
+    .getText();
   browser.getByTestId("token-amount-use-maximum").click();
 });
 
 When(
   /^I can see the maximum amount of tokens in my wallet are in the token input box$/,
   () => {
-    const noZeroes = currentAmountOfTokensInWallet.toString();
-    const noZeroesFloat = parseFloat(noZeroes);
+    const tokenBalance = currentAmountOfTokensInWallet.toString();
+    const tokenBalanceFloat = parseFloat(tokenBalance.replace(",", ""));
     expect(browser.getByTestId("token-amount-input").getValue()).toBe(
-      String(noZeroesFloat)
+      String(tokenBalanceFloat)
+    );
+  }
+);
+
+Then(/^I can see the validator node list$/, () => {
+  expect(browser.getByTestId("validator-node-list")).toBeDisplayed();
+  const nodeList = $(".node-list").$$('[data-testid="node-list-item"]');
+  expect(nodeList.length).toBeGreaterThan(0);
+});
+
+Then(/^the epoch countdown timer is counting down$/, () => {
+  const currentCountdownTimerText = browser
+    .getByTestId("current-epoch-ends-in")
+    .getText();
+  /* eslint-disable wdio/no-pause */
+  browser.pause(2100); // let some time pass
```

Comment:
Let's add a comment to that effect then

```suggestion
  browser.pause(2100); // let some time pass so the epoch timer advances
```